config/config.go

```diff
@@ -530,6 +534,17 @@ func Init() error {
 		Parameters.SigChainCacheSize = uint32(SigChainSkipMinerBlocks + 64)
 	}
 
+	// config for client relay message cache, config in percent or mega bytes. change it into bytes.
+	if Parameters.ClientMsgCacheSize <= 0 {
+		Parameters.ClientMsgCacheSize = uint32(float64(memory.TotalMemory()) * defaultClientMsgCachePercent / 100.0)
+		if Parameters.ClientMsgCacheSize == 0 {
+			Parameters.ClientMsgCacheSize = defaultClientMsgCacheSize * 1024 * 1024 // use bytes in program
+		}
+		log.Printf("Set ClientMsgCacheSize to %v", Parameters.ClientMsgCacheSize)
+	} else {
+		Parameters.ClientMsgCacheSize = Parameters.ClientMsgCacheSize * 1024 * 1024 // convert it to mega bytes.
```

Comment:
`convert it to mega bytes` -> `convert it to bytes`

Reply:
yes. modified.